src/gui/tray/UserLine.qml

```diff
@@ -81,6 +89,10 @@ MenuItem {
                             y: accountStateIndicatorBackground.y + 1
                             sourceSize.width: Style.accountAvatarStateIndicatorSize
                             sourceSize.height: Style.accountAvatarStateIndicatorSize
+
+                            Accessible.role: Accessible.Indicator
+                            Accessible.name: isConnected ? qsTr("Connected") : qsTr("Disconnected")
+                            Accessible.description: qsTr("Icon that indicates current connection state for the corresponding account")

```

Comment:
If we say "Account connected" and "Account not connected" we don’t really need the long description of "Icon that indicates current connection state for the corresponding account".

What do you think @tyrylu @Adriorjalesvidal?